.github/workflows/main.yml

```diff
@@ -91,6 +91,34 @@ jobs:
     uses: ./.github/workflows/integration-test-containers.yml
     secrets: inherit
 
+  scan-build:
+    runs-on: ubuntu-latest
+    needs: build-builder-image
+    env:
+      COLLECTOR_BUILDER_TAG: ${{ needs.build-builder-image.outputs.collector-builder-tag }}
+    steps:
+      - uses: actions/checkout@v3
+        with:
+          submodules: true
+
+      - name: Run scan build
+        run: |
+          set -x
```

Comment:
We probably don't want that in CI

Reply:
Yup, I was debugging something apparently, I'll remove it.